<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mp3encodeClass</class>
 <widget class="QDialog" name="mp3encodeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mp3encode</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resource.qrc">
    <normaloff>:/icons/resourcen/mp3encoder.png</normaloff>:/icons/resourcen/mp3encoder.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resource.qrc">
         <normaloff>:/images/resourcen/player_play.png</normaloff>:/images/resourcen/player_play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resource.qrc">
         <normaloff>:/images/resourcen/cancel.png</normaloff>:/images/resourcen/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="id3tagEditorButton">
       <property name="text">
        <string>ID3-Tag-Editor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setupButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Setup</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resource.qrc">
         <normaloff>:/images/resourcen/configure.png</normaloff>:/images/resourcen/configure.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Queue</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTreeWidget" name="QueueTreeWidget">
        <property name="dragDropMode">
         <enum>QAbstractItemView::DropOnly</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Audio Encoding</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Number of files in paths:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="numFiles">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Number of files in manual queue:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="numQueue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Audio-Profile:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="audioProfileComboBox"/>
        </item>
        <item>
         <widget class="QCheckBox" name="deleteSourceCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>delete source file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Encoding</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeWidget" name="Progress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>CPU</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Length</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Progress</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rest</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rate</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Encoding</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Done</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="Done">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Length</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rate</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Encoding</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>delete</string>
          </property>
          <property name="icon">
           <iconset resource="../../../resource.qrc">
            <normaloff>:/images/resourcen/emptytrash.png</normaloff>:/images/resourcen/emptytrash.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
